### PR TITLE
OHRM5X-1733: Develop buzz - share photo component

### PR DESCRIPTION
**Pull request by Super-Chama**

### CODE REVIEW

File: src/client/src/orangehrmBuzzPlugin/components/PhotoFrame.vue [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        'orangehrm-buzz-photos': true,
+        '--two-thumbnails': this.media.length === 2,
+        '--three-thumbnails': this.media.length === 3,
+        '--four-thumbnails': this.media.length >= 4,
+      };
+    },
+    photos() {
+      return (this.media || []).map(photo => {
+        const {type, base64} = photo;
+        return `data:${type};base64,${base64}`;
```
Comment (RajithaKumara) [inline src/client/src/orangehrmBuzzPlugin/components/PhotoFrame.vue:55]: can this component reuse, if we provide img src as URLs?

File: src/client/src/orangehrmBuzzPlugin/components/PhotoInput.vue [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    const validationMessage = ref(null);
+    const fileTypeValidator = validFileTypes([
+      'image/gif',
+      'image/jpeg',
+      'image/jpg',
+      'image/pjpeg',
+      'image/png',
+      'image/x-png',
+    ]);
+    const fileSizeValidator = maxFileSize(1024 * 1024 * 2);
```
Comment (RajithaKumara) [inline src/client/src/orangehrmBuzzPlugin/components/PhotoInput.vue:95]: will get valid file types and size from BE via props. any possibility to hard code in controller for the moment

File: src/client/src/orangehrmBuzzPlugin/components/PhotoUploadArea.vue [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+</template>
+
+<script>
+import Icon from '@ohrm/oxd/core/components/Icon/Icon';
+
+export default {
+  name: 'PhotoUploadArea',
+
+  components: {
+    'oxd-icon': Icon,
```
Comment (RajithaKumara) [inline src/client/src/orangehrmBuzzPlugin/components/PhotoUploadArea.vue:39]: shall we register Icons as a global component, if it is highly used?

File: installer/Migration/V5_3_0/lang-string/buzz.yaml [hunk 4]
```diff
@@ -53,12 +53,18 @@ langStrings:
   -
     value: 'Share'
     unitId: share
-  -
-    value: 'Share Video'
```
Comment (RajithaKumara) [inline installer/Migration/V5_3_0/lang-string/buzz.yaml:57]: ??

**Review by RajithaKumara — APPROVED** [review]

File: installer/Migration/V5_3_0/lang-string/buzz.yaml (hunk 4 above)
Comment (Super-Chama) [reply installer/Migration/V5_3_0/lang-string/buzz.yaml:57]: Already [defined](https://github.com/orangehrm/orangehrm/blob/5.3/installer/Migration/V5_3_0/lang-string/buzz.yaml#L22)

File: src/client/src/orangehrmBuzzPlugin/components/PhotoUploadArea.vue (hunk 3 above)
Comment (Super-Chama) [reply src/client/src/orangehrmBuzzPlugin/components/PhotoUploadArea.vue:39]: I think we have globally registered `oxd-icon-button` component. this is used in few places, but at this stage no difference

File: src/client/src/orangehrmBuzzPlugin/components/PhotoInput.vue (hunk 2 above)
Comment (Super-Chama) [reply src/client/src/orangehrmBuzzPlugin/components/PhotoInput.vue:95]: I checked. Issue is this componet is deep inside. maybe we will have to use provide/inject

File: src/client/src/orangehrmBuzzPlugin/components/PhotoFrame.vue (hunk 1 above)
Comment (Super-Chama) [reply src/client/src/orangehrmBuzzPlugin/components/PhotoFrame.vue:55]: yeah. planning to reuse.

**Pull request merged**